### PR TITLE
updated views for user customization

### PR DESCRIPTION
**Pull request by alr2413**
I have updated the following sections:

change env() syntax to config() syntax in viewable files.
updated route list to support middleware.
added views directory for user customization in vendor:publish command.

### CODE REVIEW

File: src/routes/web.php [hunk 1]
```diff
@@ -1,20 +1,23 @@
...
+    Route::get("$url/run/{task}", ['as' => "$url.run", 'uses' => "$controller@run"]);
 
-Route::get("$url/run/{task}", "$controller@run")->name("$url.run");
+    Route::get("$url/create", ['as' => "$url.create", 'uses' => "$controller@create"]);
 
-Route::get("$url/create", "$controller@create")->name("$url.create");
+    Route::post("$url/store", ['as' => "$url.store", 'uses' => "$controller@store"]);
 
-Route::post("$url/store", "$controller@store")->name("$url.store");
+    Route::delete("$url/delete", ['as' => "$url.delete", 'uses' => "$controller@delete"]);
```
Comment (MatviiB) [inline src/routes/web.php:24]: please do something with case if $url contains '/' and   ` 'as' `will be as "module/scheduler.delete". And I preferred when name is set as `->name('foo.bar')`. Please. Thanks

Comment (alr2413) [reply src/routes/web.php:24]: We can replace it with syntax like `str_replace('/', '.', $url)`.
is this okay?
Thanks.

Comment (MatviiB) [reply src/routes/web.php:24]: yes, that's fine

Comment (alr2413) [reply src/routes/web.php:24]: Okay.
I did that.
Thanks.

**Review by MatviiB — APPROVED** [review]

**Pull request merged**